docs/sphinx/examples/python/tutorials/Shors.ipynb

```diff
@@ -29,7 +29,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
+   "id": "367c0921",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install contfrac"
```

Comment:
I thought we determined that doing `pip install` from within the notebook didn't reliably work because we were running the notebook from within the context of `docs/notebook_validation.py`. Am I misremembering that?

Reply:
FWIW, we added `contfrac` to one of the CI images here during the PR:
https://github.com/NVIDIA/cuda-quantum/blame/c4001f66a8f90c0367126b0fdc5df59f24cd8cdf/.github/workflows/docker_images.yml#L781

So perhaps we need to also add it to the Publishing workflow somehow (perhaps as opposed to putting the `pip install` in the notebook itself).